### PR TITLE
Add versioneer

### PR DESCRIPTION
**Pull request by matthieubulte**
Hello, I was also a little bothered by #28 so this PR adds versioneer (which is the tools used for other dask projects) to automatically handle the version present in `__init__.py`.

Please let me know if there is anything to change.

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Comment by mrocklin** [comment]
@quasiben can you review and merge this if appropriate?

File: setup.cfg [hunk 1]
```diff
@@ -10,3 +10,10 @@ ignore =
     W503
     # whitespace before :
     E203
+[versioneer]
+VCS = git
+style = pep440
+versionfile_source = dask_cuda/_version.py
+versionfile_build = dask_cuda/_version.py
+tag_prefix =
+parentdir_prefix = parentdir_prefix-
```
Comment (pentschev) [inline setup.cfg]:
```suggestion
parentdir_prefix = dask_cuda-
```

Comment (matthieubulte) [reply setup.cfg]: Sorry about that, will fix.

Comment (matthieubulte) [reply setup.cfg]: It's now fixed.

**Comment by dillon-cullinan** [comment]
ok to test

**Comment by pentschev** [comment]
Thanks @matthieubulte for this PR. For future reference, you don't need to amend and force-push, we squash-merge PRs.

Also, the failures are not related to this PR, changes from 0.8 are still not reflected in 0.9, I will rerun tests and merge this as soon as #76 is merged.

**Comment by matthieubulte** [comment]
Alright, thanks!

**Comment by pentschev** [comment]
rerun tests

**Comment by pentschev** [comment]
Thanks @matthieubulte for the contribution, merging.

**Pull request merged**